extensions/wikia/Email/Controller/UserRightsChangedController.class.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Email\Controller;
+
+use Email\Check;
+use \Email\EmailController;
+
+class UserRightsChangedController extends EmailController {
+
+	const LAYOUT_CSS = 'avatarLayout.css';
```

Comment:
This shouldn't be needed here; this constant's already defined as avatarLayout.css in EmailController.